imap.py

```diff
@@ -166,6 +173,10 @@ def get_mails_cron(cls):
         Cron get mails:
         - State: active
         """
+        if not PRODUCTION_ENV:
+            logger.warning('Production mode is not enabled.')
```

Comment:
Also in get_mails() has control production mode (production mode in cron method is optional)